pkg/client/generators.go

```diff
@@ -0,0 +1,46 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/wavesplatform/gowaves/pkg/api"
```

Comment:
Do not import `api` package here. `api` package is our node implementation details, while `client` is a library.